packages/x-adapter-platform/src/mappers/responses/__tests__/search-response.mapper.spec.ts

```diff
@@ -1,8 +1,11 @@
-import { searchResponseMapper } from '../search-response.mapper';
 import { platformSearchResponse } from '../../../__tests__/__fixtures__/search.response';
+import { PlatformSearchResponse } from '../../../types';
```

Comment:
avoid importing from the barrel

```suggestion
import { PlatformSearchResponse } from '../../../types/responses/search-response.model';
```